sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/report/XrayWrapperReportReaderTest.java

```diff
@@ -24,24 +26,24 @@ class XrayWrapperReportReaderTest {
     @BeforeEach
     void beforeEach() {
         reportReader = new XrayWrapperReportReader();
-        mapper = new ObjectMapper();
+        mapper = XrayWrapperJSONConverter.get().getMapper();
     }
 
     @Test
-    void getReportFiles_get_valid_reports() {
+    void getReportFiles_get_valid_reports() throws XrayWrapperReportException {
         /* prepare */
         String source = "src/test/resources/xray-report-examples";
         String resultFile = "resultfile";
         String cycloneReport = "Docker_SBOM_Export_CycloneDX.json";
         String securityReport = "Docker_Security_Export.json";
 
         /* execute */
-        reportReader.getReportFiles(source, resultFile);
+        reportReader.findXrayReportsInArchive(source, resultFile);
 
         /* test */
         String cycloneName = reportReader.cycloneReport.getName();
         String securityName = reportReader.securityReport.getName();
-        String sechubReport = reportReader.sechubReport.getName();
+        String sechubReport = reportReader.xrayPdsReport.getName();
         assertEquals(resultFile, sechubReport);
```

Comment:
Normally I am a fan of local variables, but here I think that 
```suggestion
        assertEquals("resultfile", sechubReport);
        assertEquals("Docker_SBOM_Export_CycloneDX.json", cycloneName);
        assertEquals("Docker_Security_Export.json", securityName);
```

would be easier to read/maintain.